share/completions/killall.fish

```diff
@@ -22,7 +22,11 @@ if test (uname) != 'SunOS'
 		end
 	end
 
-	complete -c killall -xa '(__fish_complete_proc)'
+    if test $USER = root
```

Comment:
That var should always be set but it's better to be safe and put double-quotes around it. This is one place where the usual fish semantics aren't in effect in order to be POSIX compliant. If `$USER` isn't set a "missing argument" error will be thrown.